Client/Experiments/Settings/ExperimentsSettingsViewController.swift

```diff
@@ -37,7 +37,7 @@ class ExperimentsSettingsViewController: UIViewController {
     }
 
     @objc private func updateState() {
-        experimentsView.reloadButton.isEnabled = !(experimentsView.customRemoteSettingsTextField.text?.isEmpty ?? true)
+        experimentsView.reloadButton.isEnabled = (experimentsView.customRemoteSettingsTextField.text?.isNotEmpty ?? true)
```

Comment:
```suggestion
        experimentsView.reloadButton.isEnabled = !(experimentsView.customRemoteSettingsTextField.text?.isEmpty ?? true)
```

Reply:
Could be
`experimentsView.reloadButton.isEnabled = experimentsView.customRemoteSettingsTextField.text?.isNotEmpty ?? false` ?

Reply:
Personally I never like those `someOptionalBool ?? [true|false]` as they need some real thought to figure out what is happening in the nil case. I've come to preferring actually comparing the optional with the value you want:
```
experimentsView.reloadButton.isEnabled = (experimentsView.customRemoteSettingsTextField.text?.isNotEmpty == true)
```

I think early Swifts wouldn't allow that, but it's perfectly legal nowadays.